.github/workflows/issue_comment.yml

```diff
@@ -0,0 +1,16 @@
+name: Assign Reviewers
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  add_comment_labels:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Label issue
+        uses: ./
+        with:
+          token: ${{secrets.REPO_TOKEN}} # default token in GitHub Workflow
+          action: labels_by_user_input
```

Comment:
we want all of them to be enabled in this repo

Reply:
They have different triggers.
pr_actions.yml - triggered by a push/pr and assigns reviewers, labels by size.
issue_comment.yml - triggered by a new/edited comment, and labels the pr if there's need

Both in .github/workflows/